lib/json_api_controller/updatable_resource.rb

```diff
@@ -44,6 +44,10 @@ def destroy_links
       resource = controlled_resources.first
       resource_class.transaction do
         destroy_relation(resource, relation, params[:link_ids])
+        # as this resource may not have changed but the linked resources may have
+        # ensure we modify the upated_at timestamp to cache bust this resource
```

Comment:
fix my spelling mistake that was copied over 🙊 
```suggestion
        # ensure we modify the updated_at timestamp to cache bust this resource
```